Apps/FlightSoftware/Watchdog/src/main.c

```diff
@@ -54,16 +57,15 @@ void enterMode(enum rover_state newstate) {
         setFPGAReset();
         setMotorsReset();
         setHerculesReset();
-//        fuelGaugeLowPower();
+        I2C_Sensors__fuelGaugeLowPowerBlocking();
         /* TODO: do we want to do it in this order? */
 
         /* turn off voltage rails */
         disable3V3PowerRail();
         disable24VPowerRail();
-//        disableBatteries();
+        disableBatteries();

```

Comment:
I2C sensors work with batteries disabled now?

Reply:
No we just won't get any readings while in KEEPALIVE, I was told to do it this way by Raewyn & Lou. I can point you toward the slack messages if you'd like